spec.json

```diff
@@ -36,5 +74,6 @@
         "description": "File key (name) inside S3"
       }
     }
-  }
+  },
```

Comment:
I don't think json likes trailing commas. Also don't need the extra line after this.